Calibration/HcalAlCaRecoProducers/test/AlCaHBHEMuonFilterProducer_cfg.py

```diff
@@ -33,9 +38,11 @@
         ),
                                                          eventAutoFlushCompressedSize = cms.untracked.int32(5242880),
                                                          outputCommands = process.OutALCARECOHcalCalHBHEMuonFilter.outputCommands,
-                                                         fileName = cms.untracked.string('PoolOutput.root'),
+                                                         fileName = cms.untracked.string('OutputHBHEMuon.root'),
                                       )
 
+process.AlcaHBHEMuonFilter.Triggers = []
```

Comment:
I'm puzzled by this line, should sime triggers be in here? Or better: shouldnt this be governed by the trigger bit, something like https://github.com/cms-sw/cmssw/blob/master/Calibration/TkAlCaRecoProducers/python/ALCARECOSiPixelCalCosmics_cff.py#L19 and then an update to the GT?

Reply:
For tests I am not testing on triggers. Production jobs relies on them